core/client_test.go

```diff
@@ -20,7 +20,7 @@ func TestRemoteClient_Status(t *testing.T) {
 }
 
 func TestRemoteClient_StartBlockSubscription_And_GetBlock(t *testing.T) {
-	ctx, cancel := context.WithTimeout(context.Background(), time.Second*10)
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*15)
```

Comment:
Any good reason behind this change?

Reply:
This test was failing due to a timeout before this change. Locally it was taking approximately 11 seconds for the test to complete so I bumped it up to 15 to be safe.